app/assets/images/svg/middleware_datasource.svg

```diff
@@ -0,0 +1,11 @@
+<svg width="90" height="100" xmlns="http://www.w3.org/2000/svg">
```

Comment:
is the svg addition necessary? can't fa-database from https://www.patternfly.org/styles/icons/ be used instead?
cc @epwinchell for the icons part - please check the png/svg and their necessity. I have a feeling there might be very similar icons already in the repo ;)